rpcs3/rpcs3qt/game_list_frame.h

```diff
@@ -96,7 +99,8 @@ namespace category // (see PARAM.SFO in psdevwiki.com) TODO: Disc Categories
 		{ "MS", psp_save }  // data
```

Comment:
I'm adding this comment here a few lines too far down, but there are comments for "non_disc_games" in cat_boot.
You should change those accordingly